### ISSUE TITLE
enable github actions

### ISSUE THREAD

**Issue by pkra** [boolesrings/mathblogging-base#1]
Let's try and run this on GitHub.

**Comment by pkra** [comment]
This is what the server does right now

```bash
cd /var/services/homes/pkra/boolesrings-base && \
node ./app2.js && \
git -C ./boolesrings.org fetch --depth 1 && \
git -C ./boolesrings.org reset --hard origin/master &&\
node ./app.js && \
git -C ./boolesrings.org -add -u && \
git -C ./boolesrings.org commit -m"`date`" && \
git -C ./boolesrings.org push origin master && \
```

**Comment by pkra** [comment]
Honestly, this repo needs a complete rewrite...

**Comment by scoskey** [comment]
I was thinking of creating the ability to make a "slice" of mathblogging. Anyone could create a repo that specifies any subset of MB's feeds, plus some basic theme templates, paste in our action code, and then you get a nice site.

**Comment by pkra** [comment]
Good point. I think the question is from which end to start - modularize mathblogging or to create a clean "slice" here that mathblogging would build upon later. 

Roughly, the "slice" would take a list of feeds, merge them, truncate, write out HTML and RSS.
